### PR TITLE
IPFabric integration settings updates

### LINKED ISSUE #240: SSoT NAUTOBOT - IPFABRIC: django.core.exceptions.ValidationError
Hi,
I can't finish syncing NAUTOBOT (v1.6.3) with IPFABRIC (v6.3.1). The plugin NAUTOBOT-SSOT-IPFABRIC (v2.0.2) always ends with the same error.

```
Traceback (most recent call last):
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/jobs/base.py", line 337, in run
    self.sync_data()
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/jobs.py", line 285, in sync_data
    dest.sync_from(ipfabric_source)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/__init__.py", line 561, in sync_from
    result = syncer.perform_sync()
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 327, in perform_sync
    changed |= self.sync_diff_element(element)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 391, in sync_diff_element
    changed |= self.sync_diff_element(child, parent_model=dst_model)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 367, in sync_diff_element
    changed, modified_model = self.sync_model(src_model=src_model, dst_model=dst_model, ids=ids, attrs=attrs)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 424, in sync_model
    dst_model = dst_model.delete()
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/diffsync/diffsync_models.py", line 432, in delete
    self.safe_delete(
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/diffsync/diffsync_models.py", line 84, in safe_delete
    tonb_nbutils.tag_object(nautobot_object=nautobot_object, custom_field="ssot-synced-from-ipfabric")
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/utilities/nbutils.py", line 256, in tag_object
    _tag_object(nautobot_object)
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/utilities/nbutils.py", line 254, in _tag_object
    nautobot_object.validated_save()
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot/core/models/__init__.py", line 79, in validated_save
    self.full_clean()
  File "/opt/nautobot/lib64/python3.8/site-packages/django/db/models/base.py", line 1251, in full_clean
    raise ValidationError(errors)
django.core.exceptions.ValidationError: {'status': ["status instance with id UUID('cf4428a4-5d7a-45c4-b43c-0c64bccdae62') does not exist."]}

Traceback (most recent call last):
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/jobs/base.py", line 337, in run
    self.sync_data()
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/jobs.py", line 285, in sync_data
    dest.sync_from(ipfabric_source)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/__init__.py", line 561, in sync_from
    result = syncer.perform_sync()
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 327, in perform_sync
    changed |= self.sync_diff_element(element)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 391, in sync_diff_element
    changed |= self.sync_diff_element(child, parent_model=dst_model)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 367, in sync_diff_element
    changed, modified_model = self.sync_model(src_model=src_model, dst_model=dst_model, ids=ids, attrs=attrs)
  File "/opt/nautobot/lib64/python3.8/site-packages/diffsync/helpers.py", line 424, in sync_model
    dst_model = dst_model.delete()
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/diffsync/diffsync_models.py", line 432, in delete
    self.safe_delete(
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/diffsync/diffsync_models.py", line 84, in safe_delete
    tonb_nbutils.tag_object(nautobot_object=nautobot_object, custom_field="ssot-synced-from-ipfabric")
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/utilities/nbutils.py", line 256, in tag_object
    _tag_object(nautobot_object)
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot_ssot/integrations/ipfabric/utilities/nbutils.py", line 254, in _tag_object
    nautobot_object.validated_save()
  File "/opt/nautobot/lib64/python3.8/site-packages/nautobot/core/models/__init__.py", line 79, in validated_save
    self.full_clean()
  File "/opt/nautobot/lib64/python3.8/site-packages/django/db/models/base.py", line 1251, in full_clean
    raise ValidationError(errors)
django.core.exceptions.ValidationError: {'status': ["status instance with id UUID('cf4428a4-5d7a-45c4-b43c-0c64bccdae62') does not exist."]}
```

### PR DESCRIPTION
**Pull request by alhogan**
- Cleans up references to configuration settings in code
- Updates configuration settings names to ensure no conflicts and adhere to standards
- Adds additional documentation around required and optional configuration settings
- Fixes a few incorrect defaults for safe delete mode statuses
- Closes #240

### CODE REVIEW

File: development/nautobot_config.py [hunk 1]
```diff
@@ -236,10 +236,9 @@
         "infoblox_username": os.getenv("NAUTOBOT_SSOT_INFOBLOX_USERNAME"),
         "infoblox_verify_ssl": is_truthy(os.getenv("NAUTOBOT_SSOT_INFOBLOX_VERIFY_SSL", True)),
         "infoblox_wapi_version": os.getenv("NAUTOBOT_SSOT_INFOBLOX_WAPI_VERSION", "v2.12"),
-        "ipfabric_api_token": os.getenv("IPFABRIC_API_TOKEN"),
-        "ipfabric_host": os.getenv("IPFABRIC_HOST"),
-        "ipfabric_ssl_verify": is_truthy(os.getenv("IPFABRIC_VERIFY", "False")),
-        "ipfabric_timeout": int(os.getenv("IPFABRIC_TIMEOUT", "15")),
+        "ipfabric_api_token": os.getenv("NAUTOBOT_SSOT_IPFABRIC_API_TOKEN"),
```
Comment (chadell) [inline development/nautobot_config.py:239]: Please, update https://github.com/nautobot/nautobot-plugin-ssot/blob/develop/development/development.env#L97-L99

Comment (chadell) [reply development/nautobot_config.py:239]: and https://github.com/nautobot/nautobot-plugin-ssot/blob/develop/development/creds.example.env#L48

**Review by chadell — CHANGES_REQUESTED** [review]
missing development updates

File: development/nautobot_config.py (hunk 1 above)
Comment (alhogan) [reply development/nautobot_config.py:239]: Updated

**Review by chadell — APPROVED** [review]

**Pull request merged**